src/functions.jl

```diff
@@ -107,6 +113,17 @@ if VERSION >= v"1.1"
     end
 end
 
+@static if VERSION > v"1.9-DEV"
+    function Base.stack(A::KeyedArray; dims::Colon=:)
+        data = @invoke Base.stack(A::AbstractArray; dims)
```

Comment:
We should consider just depending on Compat.jl for this. `Compat = "3.46, 4.2"`

Here too there's an internal method we could call after dispatch, instead of `@invoke`:
```
stack(iter; dims=:) = _stack(dims, iter)
```

Reply:
There is also `stack.jl` BTW: https://github.com/mcabbott/AxisKeys.jl/blob/cddf60f65113796bba7ff6371112e1827fae8468/src/stack.jl#L7 

LazyStack@0.1 is much simplified, to use Base.stack where possible. Some of the methods overloaded there don't exist anymore. No pressure though to sort it all out, that version is presently marked incompatible. 

But perhaps `stack` needs to be qualified in the tests, to avoid some failures?